include/kiwixserve.h

```diff
@@ -0,0 +1,32 @@
+#ifndef KIWIXLIB_KIWIX_SERVE_H_
+#define KIWIXLIB_KIWIX_SERVE_H_
+
+#ifdef _WIN32
+// winsock2.h need to be included before windows.h (included by curl.h)
+# include <winsock2.h>
+#endif
+
+#include "subprocess.h"
```

Comment:
We must not include subprocess here. This is a private header.
You can simply declare the `Subprocess` class and include the `subprocess.h` header in the cpp file.